src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -97,12 +97,17 @@ net_device_table_mgr::net_device_table_mgr() : cache_table_mgr<ip_address,net_de
 	}
 	BULLSEYE_EXCLUDE_BLOCK_END
 
-	if (map_net_devices()) {
+	/* Read Link table from kernel and save it in local variable. */
+	update_tbl();
+	if (m_if_indx_to_nd_val_lst.empty()) {
 		ndtm_logdbg("map_net_devices failed");
```

Comment:
Please update comment according to the name of the function.
I dont agree with the change - now, when VMA will not manage to offloaded any device - the list size will be zero - but update_tbl did not fail...